### PR TITLE
Make cluster network optional for vagrant

### PR DESCRIPTION
**Pull request by rubenk**
Signed-off-by: Ruben Kerkhof <ruben@42on.com>

### CODE REVIEW

File: roles/ceph-defaults/defaults/main.yml [hunk 1]
```diff
@@ -335,7 +335,6 @@ osd_memory_target: 4294967296
 journal_size: 5120 # OSD journal size in MB
 block_db_size: -1 # block db size in bytes for the ceph-volume lvm batch. -1 means use the default of 'as big as possible'.
 public_network: 0.0.0.0/0
-cluster_network: "{{ public_network | regex_replace(' ', '') }}"
```
Comment (guits) [inline roles/ceph-defaults/defaults/main.yml:338]: why are you removing this?

Comment (guits) [reply roles/ceph-defaults/defaults/main.yml:338]: this is a breaking change for the ceph-infra role

Comment (rubenk) [reply roles/ceph-defaults/defaults/main.yml:338]: if cluster_network is defined, it ends up in /etc/ceph/ceph.conf, even if it's the same as public_network

Comment (guits) [reply roles/ceph-defaults/defaults/main.yml:338]: What is the issue with that?

This is one of the multiple possible reasons for writing a commit message actually: describing what you are changing and why...
By the way, I don't think having 2 commits is needed here, you might want to squash them.

Thanks!

Comment (rubenk) [reply roles/ceph-defaults/defaults/main.yml:338]: I don't need a cluster network, and prefer having less settings in ceph.conf. I thought that was obvious but I'll amend the commit message and squash.

**Comment by guits** [comment]
you have to run `./generate_group_vars_sample.sh`

**Pull request closed without merging**